src/routes/tools/local-climate-change-snapshot/explore.svelte

```diff
@@ -85,18 +100,27 @@
         ),
       };
       // Get params object for querying the Cal-Adapt API
+      // extra months param required for April SWE indicator
+      const months = $indicatorStore.id === "swe" ? DEFAULT_SWE_MONTH : null;
```

Comment:
@mukhtyar when looking at SWE, will the user be able to select the months? Or do they always default to the default SWE months?